templates/jlink/jlink-builder/jlink-builder-template.yaml

```diff
@@ -49,3 +58,31 @@ objects:
       to:
         kind: ImageStreamTag
         name: ubi9-openjdk-${JDK_VERSION}-jlink:latest
+- apiVersion: build.openshift.io/v1
+  kind: BuildConfig
+  metadata:
+    name: jlink-s2i-jdk-${JDK_VERSION}
+    labels:
+      app: jlink-s2i-jdk-${JDK_VERSION}
+  spec:
+    source:
+      type: Git
+        git: 
+          uri: ${APPLICATION}
+          ref: ${REF}
+        contextDir: ${CONTEXT_DIR}
+  strategy:
+    sourceStrategy:
+      from:
+        kind: ImageStreamTag
+        name: ubi9-openjdk-${JDK_VERSION}-jlink:latest # Output Imagestream for Stage 1
+        pullPolicy: Always
+      env:
+        - name: S2I_ENABLE_JLINK
+          value: true
+        - name: LOGGING_SCRIPT_DEBUG
+          value: true
+  output:
+    to:
+      kind: ImageStreamTag
+      name: ubi9-openjdk-${JDK_VERSION}-jlink:latest
```

Comment:
this is the input `imageStream`. This phase needs to output to a different `imageStream`, which will also need adding to the template. The name could include `intermediate` in it. for a first cut that would be enough.  Later it should probably also be unique to the app in some way. We might need to add a template parameter that the user can provide that we incorporate into the name. But I'd put that into another work stream and not block this PR on it.